tests/unit/FileUploadPreview.spec.ts

```diff
@@ -35,17 +35,6 @@ function myArrayBuffer () {
   })
 }
 
-/**
- * Utility method to get around with the timing issues
- */
-async function waitForUpdate (numTimesToFlushPromises) {
-  await Vue.nextTick()
-  for (let i = 0; i < numTimesToFlushPromises; i++) {
-    await flushPromises()
-  }
-  await Vue.nextTick()
-}
-
```

Comment:
I moved this to its own TS file so it can be imported in a couple of test suites (instead of duplicated).